plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.ts

```diff
@@ -154,6 +166,17 @@ export class GoogleAnalytics implements AnalyticsApi {
       return;
     }
 
+    if (this.virtualSearchPageView.mode !== 'disabled' && action === 'search') {
+      const { mountPath, queryParam } = this.virtualSearchPageView;
+      this.capture.pageview(
+        `${mountPath}?${queryParam}=${subject}`,
```

Comment:
I believe GA Site Search also a concept of site search _categories_ that might map well to the `types` (provided contextually as `searchTypes` [here](https://github.com/backstage/backstage/blob/master/plugins/search-react/src/context/SearchContext.tsx#L196-L202)).  Could be worth exposing that here too.